packages/geocoder/src/index.test.js

```diff
@@ -33,6 +37,7 @@ describe("geocoder", () => {
     .replyWithFile(200, mockResponsePath("arcgis", "suggest-response.json"))
     // reverse
     .get(`${baseArcGisPath}reverseGeocode`)
+    .twice()
```

Comment:
Didn't know about this but it's cool! Nice one

Reply:
😁